client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -613,7 +614,6 @@ const PatternAssembler = ( {
 				</PatternAssemblerContainer>
 			}
 			recordTracksEvent={ recordTracksEvent }
-			stepSectionName={ navigatorPath !== '/' ? 'pattern-selector' : undefined }
```

Comment:
This is only for tracking the back button. However, we hided the back button in the sub-steps so I think we can just remove this prop